x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/hooks/use_fetch_agents_data.tsx

```diff
@@ -175,6 +186,9 @@ export function useFetchAgentsData() {
   }>({});
 
   const [latestAgentActionErrors, setLatestAgentActionErrors] = useState<string[]>([]);
+  const [outputsByPolicyIds, setOutputsByPolicyIds] = useState<{
+    [k: string]: OutputsForAgentPolicy;
+  }>({});
```

Comment:
This one is for the indexed output values needed to display on the table. I'll gladly accept suggestions for better naming :)